api/src/util/time.ts

```diff
@@ -0,0 +1,29 @@
+import ms from 'ms';
+
+export function getCurrentTimeAsSeconds(): number {
```

Comment:
```suggestion
export function getSecondsSinceEpoch(): number {
```

epoch 기준이기 때문에 기준이 나타면 좋을 것 같아 말씀드립니다~

Reply:
리뷰 감사합니다.  https://github.com/MovieReviewComment/Mr.C/pull/82/commits/2447ff5da04a1fbde44ec0d94c0fb69cc1b7d758 에서 반영했습니다.